docs/components/dartpad/progress_indicators/theme/main.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(App());
+}
+
+class App extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Theme Progress Indicator Demo',
+      debugShowCheckedModeBanner: false,
+      theme: _buildShrineTheme(),
+      home: Scaffold(
+        appBar: AppBar(
+          title: Text('Theme Progress Indicator Demo'),
+        ),
+        body: Center(
+          child: Column(
+            children: [
```

Comment:
Update this demo to remove the `Divider` and instead put some space between the linear and circular

Reply:
Done